libs/utils/env.py

```diff
@@ -644,6 +644,17 @@ def _init_platform(self):
         # Adding topology information
         self.platform['topology'] = self.topology.get_level("cluster")
 
+        # Adding kernel build information
+        kver = self.target.execute('uname -r').strip()
+        match = KVERSION_RE.match(kver)
+        if match:
+            self.platform['kernel_version'] = match.group('ver')
```

Comment:
I'm using a dict because I need to dump it as a JSON... the new KernelVersion set of attributes reports separate attributes for version.major.minor... we can use these attributes to build whatever tuple we need at client side.